README.md

```diff
@@ -49,7 +49,7 @@ Got questions? Join us in [#redisearch on the Redis Discord](https://discord.gg/
 
 * Full-Text indexing of multiple fields in Redis hashes
 * Incremental indexing without performance loss
-* Document ranking (using [tf-idf](https://en.wikipedia.org/wiki/Tf%E2%80%93idf), with optional user-provided weights)
+* Document ranking (using [BM25](https://redis.io/docs/latest/develop/interact/search-and-query/advanced-concepts/scoring/) as default, with optional user-provided weights)
```

Comment:
maybe put the link to BM25 in Wikipedia instead of the idf one, and say that there are additional scoring methods available (and attach the doc link)